spec/models/related_identifier_spec.rb

```diff
@@ -104,9 +104,28 @@
         allow(Rails.logger).to receive(:info)
 
         expect(RelatedIdentifier.push_item(item)).to eq(1)
-        expect(Maremma).to have_received(:post).twice
+        expect(Maremma).to have_received(:post).with("https://fake.lagattino.com/events", anything).once
 
         expect(Rails.logger).to have_received(:info).with("[Event Data] https://doi.org/10.1234/example example_type https://doi.org/10.5678/related pushed to Event Data service.")
+      end
+
+      it "does not push the event to the event data bus when source_id is not datacite_crossref" do
```

Comment:
I think this test is misnamed, as it checks that the event gets posted to _both_ the internal Event Data on Lupo and also to the Bus. Not even anything close to a blocker, but fixing it now might make thins easier in the future :sweat_smile:

Reply:
Good catch @digitaldogsbody. I've updated the pr 👍🏽